<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page2</class>
 <widget class="QMainWindow" name="page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>891</width>
      <height>851</height>
     </rect>
    </property>
    <property name="title">
     <string>sign in</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="namelineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>Enter your name...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passlineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter your password...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logpushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>520</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>650</y>
       <width>371</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="randlineEdit_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>151</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="enterNumLineEdit_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>410</y>
       <width>181</width>
       <height>48</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>410</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/icons8-refresh-48.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>520</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>I'm not a company</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idlineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>Enter your id...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>260</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delnamelineEdit">
    <property name="geometry">
     <rect>
      <x>992</x>
      <y>100</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
